schema/test/schema.test.ts

```diff
@@ -126,6 +138,19 @@ describe("Validate manifests", () => {
     expect(valid).toBe(false);
   });
 
+  it("accepts a local CSP connect-src without a purpose", () => {
+    const valid = validateLocal({
+      ...basicManifest,
+      ui_extension: {
+        ...basicManifest.ui_extension,
```

Comment:
I think we want to remove this spread from the test — this pulls in `actions` and `views`, which we should be able to omit in the local manifest.

Reply:
Done!